core-bundle/src/Controller/ContentElement/PlayerController.php

```diff
@@ -57,7 +57,7 @@ protected function getResponse(FragmentTemplate $template, ContentModel $model,
     {
         // Find and order source files
         $filesystemItems = FilesystemUtil::listContentsFromSerialized($this->filesStorage, $model->playerSRC ?: '');
-        $isVideo = \in_array($filesystemItems->first()?->getExtension(true), self::VIDEO_TYPES, true);
+        $isVideo = $filesystemItems->first()?->isVideo() ?? false;
```

Comment:
Now that we delegate the type detection to the VFS, would it make sense to do the same with the sorting further down in `getSourceFiles()` and remove the `VIDEO_TYPES`/`AUDIO_TYPES` class  constants completely?

We would probably need to add a `SortMode::mediaTypePriority` or something that you could use in `FilesystemItemIterator#sort()`.

Reply:
Implemented in 6c6fd9cb2f95f8a108ba7a8e17e09eb70c4f8a5e.

Reply:
Awesome! :heart: